packages/react-ui/src/app/features/builder/ai-chat/lib/step-settings-ai-chat-hook.ts

```diff
@@ -125,3 +132,26 @@ export const useStepSettingsAiChat = (
     isEmpty: !messages.length,
   };
 };
+
+const CODE_BLOCK_NAME = '@openops/code';
+const CODE_ACTION_NAME = 'code';
```

Comment:
We don't have a block and an action name because Code is a frontend-defined block (like switch/branch)